apps/meteor/ee/app/livechat-enterprise/server/api/priorities.ts

```diff
@@ -64,3 +64,23 @@ API.v1.addRoute(
 		},
 	},
 );
+
+API.v1.addRoute(
+	'livechat/priority.reset',
+	{
+		authRequired: true,
+		permissionsRequired: {
+			POST: { permissions: ['manage-livechat-priorities'], operation: 'hasAny' },
+			GET: { permissions: ['manage-livechat-priorities'], operation: 'hasAny' },
+		},
+	},
+	{
+		async post() {
+			await LivechatPriority.resetPriorities();
+			return API.v1.success();
```

Comment:
Do you think checking here again if priorities can be reset makes sense? Would help to prevent the update

Reply:
idk, the update is so lightweight with 5 members